src/Filter/FilterProcessor.php

```diff
@@ -0,0 +1,199 @@
+<?php
+/**
+ * This file is part of the event-engine/php-postgres-document-store.
+ * (c) 2019-2021 prooph software GmbH <contact@prooph.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace EventEngine\DocumentStore\Postgres\Filter;
+
+use EventEngine\DocumentStore;
+use EventEngine\DocumentStore\Filter\Filter;
+use EventEngine\DocumentStore\Postgres\Exception\InvalidArgumentException;
+use EventEngine\DocumentStore\Postgres\Exception\RuntimeException;
+
+/**
+ * Default filter processor class for converting a filter to a where clause.
+ */
+final class FilterProcessor implements FilterProcessorInterface
```

Comment:
Since we don't use suffixes across Event Engine repos, I'd suggest to rename the default implementation to:
`FilterProcessor` -> `PostgresFilterProcessor`
`FilterProcessorInterface` -> `FilterProcessor`

Reply:
:+1: